aospy_synthetic/calc.py

```diff
@@ -125,13 +86,32 @@ def __init__(self, proj=None, model=None, run=None, ens_mem=None, var=None,
         self.end_date_xray = tm.apply_year_offset(self.end_date)
 
         self.backend = backend
-        self.db_on = db_on
+        self.db_tracking = db_tracking
 
 
 class Calc(object):
     """Class for executing, saving, and loading a single computation."""
 
-    ARR_XRAY_NAME = 'aospy_result'
+    def __hash__(self):
+        self.file_name = self._file_name(self.dtype_out_time)
+        if self.region:
+            return hash((str(type(self)), self.file_name,
+                         self.region.name, self.run, self.var))
+        else:
+            return hash((str(type(self)), self.file_name, self.run, self.var))
+
+    def track(self):
```

Comment:
This could use a docstring.  Not self-evident from code/name what's going on (at least from first glance).